_includes/02-image.md

```diff
@@ -0,0 +1 @@
+Replace this with an image, like your photo. Ensure you include some alt-text.
```

Comment:
![Image of Klee](https://i.pximg.net/img-original/img/2021/01/28/22/23/34/87366393_p0.png)